frame/support/src/traits/dispatch.rs

```diff
@@ -56,7 +56,7 @@ pub trait OriginTrait: Sized {
 	type Call;
 
 	/// The caller origin, overarching type of all pallets origins.
-	type PalletsOrigin;
+	type PalletsOrigin: Clone;
```

Comment:
Required by benchmarks. But I think this is also a reasonable requirement.

Reply:
I really don't like adding this just for benchmarks. There should be a way around it. Couldn't you just reconstruct the thing from the account id instead of cloning?